### DISCUSSION TITLE
saf-training/courses/user/07

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#42] [Ideas]
# saf-training/courses/user/07

The MITRE Security Automation Framework Team's Introduction to InSpec profile development

https://mitre.github.io/saf-training/courses/user/07.html

**Comment by karikarshivani** [comment]
Should we considering modifying inputs after visualizing? That way they can look at failed controls, look at associated inputs, and modify inputs for failed controls? We can also then use the compare view to show progress.
